<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartLevelDialogue</class>
 <widget class="QWidget" name="StartLevelDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Revolutionary</string>
  </property>
  <widget class="QPushButton" name="startLevel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>20</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Start Button</string>
   </property>
  </widget>
  <widget class="QPushButton" name="context">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>501</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>20</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Level Background Information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="popUp">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="levelSelect">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>501</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>20</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Levels</string>
   </property>
  </widget>
  <zorder>levelSelect</zorder>
  <zorder>startLevel</zorder>
  <zorder>context</zorder>
  <zorder>popUp</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
